test/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.16
 require (
 	github.com/Microsoft/go-winio v0.4.17
 	github.com/Microsoft/hcsshim v0.8.16
-	github.com/containerd/containerd v1.5.1
+	github.com/containerd/containerd v1.5.2
```

Comment:
Why did this get updated somehow?

Reply:
Ahh you probably revendored the shim?

Reply:
yep, since I removed the old internal `debug` package